ansible/vars/defaults.yml

```diff
@@ -13,3 +13,8 @@ git_repo_versions:
   discover: master
   liquid_core: master
   setup: master
+  web_ui: master
+
+web_ui_override: false
```

Comment:
So how am I supposed to override the web UI? Do I need to specify both port and path?

Reply:
Yes.